org.caveman250.gtkeddit.json

```diff
@@ -0,0 +1,163 @@
+  "modules": [
+    {
+      "name": "mm-common",
+      "cleanup": [ "/" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/mm-common/1.0/mm-common-1.0.3.tar.xz",
+          "sha256": "e81596625899aacf1d0bf27ccc2fcc7f373405ec48735ca1c7273c0fbcdc1ef5"
+        }
+      ]
+    },
+    {
+      "name": "sigc++",
+      "config-opts": [ "--disable-documentation" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/libsigc++/3.0/libsigc++-3.0.7.tar.xz",
+          "sha256": "bfbe91c0d094ea6bbc6cbd3909b7d98c6561eea8b6d9c0c25add906a6e83d733"
+        }
+      ]
+    },
+    {
+      "name": "glibmm",
+      "config-opts": [],
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.68/glibmm-2.68.1.tar.xz",
+          "sha256": "6664e27c9a9cca81c29e35687f49f2e0d173a2fc9e98c3428311f707db532f8c"
+        }
+      ]
+    },
+    {
+      "name": "cairomm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://cairographics.org/releases/cairomm-1.16.1.tar.xz",
+          "sha256": "6f6060d8e98dd4b8acfee2295fddbdd38cf487c07c26aad8d1a83bb9bff4a2c6"
+        }
+      ]
+    },
+    {
+      "name": "pangomm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.49/pangomm-2.49.1.tar.xz",
+          "sha256": "a2272883152618fddea016a62f50eb23b9b056ab3c08f3b64422591e6a507bd5"
+        }
+      ]
+    },
+    {
+      "name": "atkmm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/atkmm/2.36/atkmm-2.36.1.tar.xz",
+          "sha256": "e11324bfed1b6e330a02db25cecc145dca03fb0dff47f0710c85e317687da458"
+        }
+      ]
+    },
+    {
+      "name": "gtkmm",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/gtkmm/4.2/gtkmm-4.2.0.tar.xz",
+          "sha256": "480c4c38f2e7ffcf58f56bb4b4d612f3f0cac9fd5908fd2cd8249fe10592a98b"
+        }
+      ]
+    },
+    {
+      "name": "jsoncpp",
+      "buildsystem": "meson",
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/open-source-parsers/jsoncpp",
+          "tag": "1.9.4"
+        }
+      ]
+    },
+    {
+      "name": "libmicrohttpd",
+      "sources":
+      [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.73.tar.gz",
+          "sha256": "a37b2f1b88fd1bfe74109586be463a434d34e773530fc2a74364cfcf734c032e"
+        }
+      ]
+    },
+    {
+      "name": "boost",
```

Comment:
I know you probably copy-pasted it from somewhere but it can be done better this way these days: https://github.com/flathub/com.prusa3d.PrusaSlicer/blob/1b9c2ae1ae723e42d86d6f83bdd007fc8a6b5600/dependencies/boost.json

Reply:
Also ideally, you should build only libraries your app is using, unless it only uses the static ones.

Reply:
fixed, only copies the headers now.